### ISSUE TITLE
pysb.readthedocs.org is out of date - webhook broken?

### ISSUE THREAD

**Issue by alubbock** [pysb/pysb#215]
I see we have readthedocs webhook intergration set up on github, but it doesn't appear to be updating. The documentation on http://pysb.readthedocs.org at the time of writing is from commit 180bf19, dating from 2015-09-29.

**Comment by jmuhlich** [comment]
Ugh, I could have sworn I made sure the build was working when I released v1.1.0. Looks like sphinx-build is failing to read (i.e. execute) doc/conf.py due to this exception:

```
File "conf.py", line 59, in <module>
  version = pkg_resources.parse_version(release).base_version
AttributeError: 'tuple' object has no attribute 'base_version'
```

It looks like in setuptools (which provides pkg_resources) before v8.0, pkg_resources.parse_version returned a tuple, but as of v8.0 it returns a custom class (which is what my code in conf.py is expecting). RTD must have a pre-8.0 version installed. I suspect adding the line `setuptools>=8` to doc/requirements.txt will fix this. The setuptools major version changes pretty quickly (presumably indicating breaking API changes?) so maybe we should just be explicit and say `setuptools>=23,<24` (the current major version).

**Comment by jmuhlich** [comment]
A little further explanation: The next commit on master after 180bf19 was when I introduced Versioneer and deleted the `get_version` function in setup.py, but forgot to update doc/conf.py which was still calling `get_version`. Then very recently I fixed that issue so at least the docs did build on my local machine, but due to the old setuptools in the RTD default environment it's still broken there.
